package.json

```diff
@@ -15,9 +15,8 @@
     "test": "jest --forceExit --coverage --verbose",
     "watch:test": "yarn test --watchAll",
     "build:ts": "etsc",
-    "build:css": "postcss public/css/tailwind.css -o dist/public/css/styles.css",
-    "build:css:prod": "NODE_ENV=production yarn build:css",
-    "watch:css": "yarn build:css -w",
+    "build:css": "NODE_ENV=production postcss public/css/tailwind.css -o dist/public/css/styles.css",
+    "watch:css": "NODE_ENV=development postcss public/css/tailwind.css -o dist/public/styles.css -w",
```

Comment:
Oh no! There's the problem I am facing! :rotating_light: It should be:
```json
"watch:css": "NODE_ENV=development postcss public/css/tailwind.css -o dist/public/css/styles.css -w",
```

Reply:
The `/css` is missing in the output path

Reply:
You're right, and I think this was causing the `watch` script issue mentioned.